virtcontainers/api.go

```diff
@@ -589,7 +589,7 @@ func statusContainer(sandbox *Sandbox, containerID string) (ContainerStatus, err
 				State:       container.state,
 				PID:         container.process.Pid,
 				StartTime:   container.process.StartTime,
-				RootFs:      container.config.RootFs,
+				RootFs:      container.config.RootFs.Target,
```

Comment:
`container.config.RootFs.Target` is not set. So container `RootFs` is always empty.